server/src/handlers.py

```diff
@@ -14,26 +14,30 @@ async def ping(request: web.Request) -> web.Response:
 
 async def create_room(request: web.Request) -> web.Response:
     data = await request.json()
-    some_room = room.Room(data)
+    room = Room(data)
     try:
-        some_room.validate()
+        room.validate()
     except DataError as e:
         return web.json_response(e.to_primitive(), status=400)
-    if await db.room_is_occupied(data):
+    pool = request.app['pool']
+    try:
+        room_id = await db.create_room(pool, room)
+    except UniqueViolationError:
         return web.json_response({'msg': 'room with that name already exists'}, status=409)
-    await db.create_room(data)
-    return web.json_response({'msg': 'OK'}, status=201)
+    return web.json_response({'id': room_id}, status=201)
 
 
 async def get_list_rooms(request: web.Request) -> web.Response:
+    pool = request.app['pool']
     page = int(request.query.get('page', '1'))
```

Comment:
IMHO, page and limit need to be validated, in case of not numeric value (400 status code)